### ISSUE TITLE
Is this still works ? If so, I want to fork it and try to add captcha bypass, and enhance it, as far as I can.

### ISSUE THREAD

**Issue by gensart-ai** [GaspardCulis/elevenlabs-unleashed#17]

**Comment by GaspardCulis** [comment]
[This guy](https://github.com/GaspardCulis/elevenlabs-unleashed/issues/15#issuecomment-1961537942) got it working. Eventually I'll update the repo, I'm currently too busy on other projects

**Comment by gensart-ai** [comment]
Yeah, I've tried to test your app, in my latest Chrome (122.x), seems like Chrome Driver now did not release anymore for version 155 or newer. I think you may need to give that info in README.

**Comment by GaspardCulis** [comment]
How the fuk did you got your hands on Chrome v155 wtf. Mine is on 122 so it works fine. Also not my job to teach people how to use selenium in the README

**Comment by gensart-ai** [comment]
Sorry, bad typo, I mean 114/115.

In your case 122 is work ? Which chromedriver you use ?

**Comment by GaspardCulis** [comment]
The one in the [AUR](https://aur.archlinux.org/).

**Comment by gensart-ai** [comment]
well, that's more clear and explainable, thanks.
